tests/integration/datadict/submission/test_endpoints.py

```diff
@@ -691,8 +703,10 @@ def test_can_submit_with_asterisk_tsv(client, pg_driver, cgci_blgsp, submitter):
     assert resp.status_code == 200, resp.data
 
 
-def test_export_entity_by_id(client, pg_driver, cgci_blgsp, submitter):
-    post_example_entities_together(client, submitter)
+def test_export_entity_by_id(
+    client, pg_driver, cgci_blgsp, submitter, require_index_exists_off
+):
+    post_example_entities_together(client, submitter, extended_data_fnames)
```

Comment:
i think duplicating this test but exporting the `experimental_metadata` node instead would help fixing the bug